reverse_strand.py

```diff
@@ -73,7 +72,9 @@
         for i in range(begin, begin + cig_length):
             cig_list.append(cig_type)
         begin += cig_length
-    for operator in range(0, int(flank_length)+2+local_region_size):
+    # Deletions need to be counted in the new variant position:
+    read_var_pos = start + int(flank_length) + 1 + cig_list.count(pysam.CDEL)
+    for operator in range(0, int(flank_length) + 2 + local_region_size + cig_list.count(pysam.CDEL)):
```

Comment:
One minor comment: there is no need to always convert `flank_length` to `int` type, you can specify it in the argument parser:

``` python
parser.add_argument("-f", "--flankingseqlength", type=int, help="length of each of the flanking sequences")
```

Also the same for `score_perc` and `diff_AS_XS_perc` throughout the code